datumaro/tests/test_transforms.py

```diff
@@ -14,29 +14,22 @@
 
 class TransformsTest(TestCase):
     def test_reindex(self):
```

Comment:
On my machine this test works. But I have another test in this section, that has stopped to work when I have rewritten it: "test_remap_labels"